### ISSUE TITLE
Error message is cryptic when you do not set attributes as an empty object while creating a schema

### ISSUE THREAD

**Issue by apandays** [cedar-policy/cedar#263]
### Before opening, please confirm:

- [X] I have [searched for duplicate or closed issues](https://github.com/cedar-policy/cedar/issues?q=is%3Aissue+).
- [X] I have read the guide for [submitting bug reports](https://github.com/cedar-policy/cedar/blob/main/CONTRIBUTING.md#bug-reports).
- [X] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

### Bug Category

Schemas and Validation

### Describe the bug

Error message is cryptic when you do not set attributes as an empty object while creating a schema. 

### Expected behavior

There are 2 solutions here 

1. Schema should be saved even if we do not specify a attributes object. Assume it to be empty 
2. Use the full term Attributes instead of Attr, in attributes

### Reproduction steps

Try to save a entity in the schema without specifying Attributes 

### Code Snippet

```rust
// Put your code below this line.

```

### Log output

```
// Put your output below this line

```

**Comment by john-h-kastner-aws** [comment]
Can you comment with the exact schema triggering this error message? This may be resolved by an unreleased change already in our `main` branch.

**Comment by john-h-kastner-aws** [comment]
Closing as fixed by #132
